packages/components/pagination/src/pagination.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PageNavigator from './page-navigator';
+import PageSizeSelector from './page-size-selector';
+import styled from '@emotion/styled';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+function Pagination(props) {
+  const totalPages = Math.ceil(props.totalItems / props.pageSize);
+
+  const currentPageItems =
+    props.currentPage === totalPages
+      ? props.totalItems - props.pageSize * (props.currentPage - 1)
+      : props.pageSize;
+
+  return (
+    <Container>
+      <PageSizeSelector
+        options={props.pageSizeOptions}
+        pageSize={props.pageSize}
+        currentPageItems={currentPageItems}
+        onPageSizeChange={props.onPageSizeChange}
+      />
+      <PageNavigator
+        totalPages={totalPages}
+        currentPage={props.currentPage}
+        onPageChange={props.onPageChange}
+      />
+    </Container>
+  );
+}
+Pagination.displayName = 'Pagination';
+Pagination.propTypes = {
+  /**
+   * Number of items per page
+   */
+  pageSize: PropTypes.number.isRequired,
```

Comment:
I would use `oneOf` here.
At least React will set a warn on the log if the wrong value is provided